src/init.rs

```diff
@@ -96,10 +96,10 @@ starship_precmd() {
     if [[ $STARSHIP_START_TIME ]]; then
         STARSHIP_END_TIME=$(date +%s);
         STARSHIP_DURATION=$((STARSHIP_END_TIME - STARSHIP_START_TIME));
-        PS1="$(starship prompt --status=$STATUS --jobs="$(jobs -p | wc -l)" --cmd-duration=$STARSHIP_DURATION)";
+        PS1="$(STARSHIP_SHELL="bash" starship prompt --status=$STATUS --jobs="$(jobs -p | wc -l)" --cmd-duration=$STARSHIP_DURATION)";
```

Comment:
I think it would be easier and cleaner to set this in the start of the script with an export line, e.g.

```
export STARSHIP_SHELL="bash";
```

at the end of the init script. Just remember to put the semicolon in, or the shell will throw up a huge fuss.